Client/Frontend/Browser/FindInPageBar.swift

```diff
@@ -104,41 +108,33 @@ class FindInPageBar: UIView, ThemeApplicable {
 
         addSubviews(searchText, matchCountView, previousButton, nextButton, closeButton, topBorder)
 
-        searchText.snp.makeConstraints { make in
-            make.leading.top.bottom.equalTo(self).inset(UIEdgeInsets(top: 0, left: 8, bottom: 0, right: 0))
-        }
-        searchText.setContentHuggingPriority(.defaultLow, for: .horizontal)
-        searchText.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        NSLayoutConstraint.activate([
+            searchText.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 8),
+            searchText.topAnchor.constraint(equalTo: topAnchor),
+            searchText.bottomAnchor.constraint(equalTo: bottomAnchor),
 
-        matchCountView.snp.makeConstraints { make in
-            make.leading.equalTo(searchText.snp.trailing)
-            make.centerY.equalTo(self)
-        }
-        matchCountView.setContentHuggingPriority(.defaultHigh, for: .horizontal)
-        matchCountView.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+            matchCountView.leadingAnchor.constraint(equalTo: searchText.trailingAnchor),
+            matchCountView.centerYAnchor.constraint(equalTo: centerYAnchor),
 
-        previousButton.snp.makeConstraints { make in
-            make.leading.equalTo(matchCountView.snp.trailing)
-            make.size.equalTo(self.snp.height)
-            make.centerY.equalTo(self)
-        }
+            previousButton.leadingAnchor.constraint(equalTo: matchCountView.trailingAnchor),
+            previousButton.widthAnchor.constraint(greaterThanOrEqualTo: heightAnchor),
+            previousButton.centerYAnchor.constraint(equalTo: centerYAnchor),
 
-        nextButton.snp.makeConstraints { make in
-            make.leading.equalTo(previousButton.snp.trailing)
-            make.size.equalTo(self.snp.height)
-            make.centerY.equalTo(self)
-        }
+            nextButton.leadingAnchor.constraint(equalTo: previousButton.trailingAnchor),
+            nextButton.widthAnchor.constraint(greaterThanOrEqualTo: heightAnchor),
+            nextButton.centerYAnchor.constraint(equalTo: centerYAnchor),
 
-        closeButton.snp.makeConstraints { make in
-            make.leading.equalTo(nextButton.snp.trailing)
-            make.size.equalTo(self.snp.height)
-            make.trailing.centerY.equalTo(self)
-        }
+            closeButton.leadingAnchor.constraint(equalTo: nextButton.trailingAnchor),
+            closeButton.widthAnchor.constraint(greaterThanOrEqualTo: heightAnchor),
```

Comment:
closeButton needs to be fixed so we use equalTo not greaterThanOrEqualTo, and the equivalent of size from SnapKit in native constraints is: widthAnchor and heightAnchor.